misc/scripts/install-local.sh

```diff
@@ -308,6 +308,10 @@ for i in "${!source[@]}"; do
     if [[ -n $PACSTALL_PAYLOAD && ! -f "/tmp/pacstall-pacdeps-$PACKAGE" ]]; then
         dest="${PACSTALL_PAYLOAD##*/}"
     fi
+    if [[ $url != *://* ]]; then
+        # shellcheck disable=SC2031,SC2086
+        url="$(< ${STGDIR}/repo/pacstallrepo)/packages/${pkgname}/${url}"
```

Comment:
This can have multiple values, each newline separated.

Reply:
We discussed in discord about this.

[`4ccf74a` (#276)](https://github.com/pacstall/pacstall/pull/276/commits/4ccf74aaa01b37e5f969d1fb6a39d5c87538040e#diff-3cfe92392f9c3534c9589d9f1f2096e7fbe6d7530ae7eae7312e72bc340ee843R126-R128).

What happens is it's overwriting the array every time, so the fix is to use:

```bash
mapfile -t REPOS -O${#REPOS[@]} <<< "${URLLIST[$IDX]}"
```

Reply:
No REPO and REPOS are not the same. This is an unrelated merge.

Reply:
See: https://github.com/pacstall/pacstall/pull/1049